outputs/alertmanager.go

```diff
@@ -16,6 +16,17 @@ type alertmanagerPayload struct {
 	EndsAt      time.Time         `json:"endsAt,omitempty"`
 }
 
+var defaultSeverityMap = map[types.PriorityType]string{
```

Comment:
why don't you use directly a `map[string]string`? It will avoid to use `String()` in line 96

Reply:
I changed the Type of the severity map of the Alertmanager Config struct to match the behavior and get ride of the .String().